src/screens/SurveyDetail/question.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import Rating from 'components/Rating';
+import NetPromotingScore from 'components/NetPromotingScore';
```

Comment:
@rafayet-monon As I comment here (https://github.com/rafayet-monon/surveyor/pull/20/files#r544779215), I think once we improve that part then we don't have to remove the the previous codebase like this ✨